rag/advanced_rag/harness/orchestrator/agentic.py

```
@@ -0,0 +1,241 @@
+            CLAIM_RESEARCH_TIMEOUT_SECONDS,
+            _snip(claim.description),
+        )
+        return {
+            "report": "",
+            "is_verified": False,
+            "confidence": 0.0,
+            "evidence_ids": [],
+            "gaps": [f"claim research timeout after {CLAIM_RESEARCH_TIMEOUT_SECONDS}s"],
+            "discovered_claims": [],
+        }
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)